server/cmwell-data-tools-app/src/main/resources/logback.xml

```diff
@@ -107,15 +107,14 @@
 
     <appender name="ASYNC_BAD_DATA" class="ch.qos.logback.classic.AsyncAppender">
         <appender-ref ref="BAD_DATA" />
-        <maxFlushTime>0</maxFlushTime>
```

Comment:
The assumption is that no log line should be discarded.
Why did you remove the unlimited time waited for log line to be printed?

Reply:
Agree done.